server/interfaces/learnerGoals.ts

```diff
@@ -5,8 +5,3 @@ export interface LearnerGoals {
   longTermGoals: string[]
   shortTermGoals: string[]
 }
-
-export interface LearnerGoalsTestParams {
```

Comment:
This is the test params object that I deleted earlier.
Notice how it was declared in `server/....` which would imply main runtime code ; but it was only used in a single cypress test (ie. this type should have been defined closer to the cypress test somewhere)